### PR TITLE
Add Search Building module

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add module to make a group search a building

### CODE REVIEW

File: addons/modules/stringtable.xml [hunk 1]
```diff
@@ -1822,6 +1822,9 @@
         <Key ID="STR_ZEN_Modules_EditableObjectsOnly_Tooltip">
             <English>Only objects that are currently editable by the local curator will be exported.</English>
         </Key>
+        <Key ID="STR_ZEN_Modules_SearchBuilding">
+            <English>Search Building</English>
```
Comment (bux) [inline addons/modules/stringtable.xml:1826]:
```suggestion
            <English>Search Building</English>
            <German>Gebäude durchsuchen</German>
```

File: addons/modules/stringtable.xml [hunk 2]
```diff
@@ -1972,6 +1975,9 @@
             <Chinesesimp>最近的房子离太远了</Chinesesimp>
             <Chinese>最近的房子離太遠了</Chinese>
         </Key>
+        <Key ID="STR_ZEN_Modules_NoBuildingFound">
+            <English>Unable to find a building to search</English>
```
Comment (bux) [inline addons/modules/stringtable.xml:1979]:
```suggestion
            <English>Unable to find a building to search</English>
            <German>Kein Gebäude zum Durchsuchen in der Nähe</German>
```

**Comment by bux** [comment]
If the group gets engaged while searching the building will they continue to search the building until no unit is left in the group?

**Comment by mharis001** [comment]
Yes, the group should continue to search until all positions have been searched or no units remain.

**Comment by bux** [comment]
Would it make sense to cancel the task if a certain amount of units (e.g. >60%) have been killed in that group? Or add an option to do that?

**Comment by mharis001** [comment]
Can look into improving the search behaviour later. Merging to include in 1.8.0.

**Pull request merged**